justspacesapp/views.py

```diff
@@ -24,13 +25,23 @@ class CreateStudy(CreateView):
     fields = '__all__'
     success_url = '/'
 
-class CreateSurvey(TemplateView):
-    # to create a sample area:
-    # INSERT INTO pldp_studyarea (name, area) VALUES ('Example area', ST_SetSRID(ST_MakePoint(1,2),4326));
-
-    template_name = "create_survey.html"
-
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        context['form'] = CreateSurveyForm()
-        return context
+# class CreateSurvey(TemplateView):
```

Comment:
The Create Form view is from fobi, including the "name" and "is_public" fields. Could you say more about how you're envisioning this relationship being documented? Ie, how what works?

I've made modifications to the Fobi Create Form Form to take out fields, and I think these can be further modified to bring it in line with the needs of the repo: https://github.com/datamade/django-fobi/pull/1

Reply:
Sure thing. We gain a lot from Fobi, and it would be useful to future maintainers of this repo (and me!) to explain how we leverage that tool. 

I think the initial pass of such an explanation should be in the README of this PR.

I could see a header in the README with something like, "Fobi views," and then explain which views Fobi gives us and link to relevant documentation. It can be concise! The aim: making it easy for newcomers to navigate your implementation of Fobi. 

Does that seem reasonable?

***
In addition, could you explain (for my own education!) what `is_public` means in Fobi? and how that relates to Just Spaces?

Reply:
On it, I'll create a section in the README! I think that will be helpful for me to have to articulate as well. 

Currently, `is_public` is just a saved Boolean attached to a survey and isn't used anywhere in the app. It's built into Fobi as a value on the [FormEntry model](https://github.com/datamade/django-fobi/blob/just-spaces-edits/src/fobi/models.py#L364) (that is, the object you create for a new form). I haven't hidden it because I thought we might use it, but I've hidden others and could do the same. There's still work to be done to solidify the starting values associated with a survey—maybe I can prioritize that tomorrow.